### PR TITLE
feat: added CA Page

### PR DESCRIPTION
**Pull request by jayeshxn**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [ ] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Description

Please describe your pull request.

❤️Thank you!

### Post merge checklist

- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by jayeshxn** [comment]
@Shurtu-gal Please Review

File: src/pages/ca.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable max-len */
+import React from 'react';
+import CAProgram from '../components/CAProgram/CAProgram';
+import { PrivateRoute } from '../components';
+
+const CA = () => (
+  <PrivateRoute>
```
Comment (Shurtu-gal) [inline src/pages/ca.js]: I don't think privateRoute is required here as logon is not necessary to access the page.

File: src/pages/ca.js
Comment (Shurtu-gal) [inline src/pages/ca.js:1]: nit: The page name can be changed. https://inno.nitrkl.in/ca doesn't look good. 🤪

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

File: src/pages/ca.js (hunk 1 above)
Comment (jayeshxn) [reply src/pages/ca.js]: It is mentioned as part of CA Registration process, that one needs to be logged in before CA Registration

File: src/pages/ca.js
Comment (jayeshxn) [reply src/pages/ca.js:1]: Any suggestions?

**Review by Shurtu-gal — APPROVED** [review]
LGTM 🤖

**Pull request merged**